include/common.h

```diff
@@ -69,6 +69,8 @@ struct nvnc_client {
 	struct nvnc_common common;
 	int ref;
 	struct stream* net_stream;
+	char hostname[128];
```

Comment:
According to a quick search, I found that the maximum name of a host name is 254 characters. So, best set this to 256.